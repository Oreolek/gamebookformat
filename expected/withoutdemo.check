examples/withoutdemo.debug: Could not reach section 'demoinfo' from start.
