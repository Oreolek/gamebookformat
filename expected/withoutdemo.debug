BEGIN DEBUG OUTPUT
Book title: Gamebook
Number of sections: 8

Turn to 1 to begin.
1 (start) -  This gamebook contains a few sections tagged as demo, that will be included when building the example as withdemo.gamebook (see withdemo.gamebook.options file). There are also some sections without the demo tag that will be ignored. This version is built without options and includes all sections. Go on to 7 or 5 or 8. 
2 (altend) -  Alternative end. Not included in the demo.
3 (demoinfo) -  This is where references to non-demo sections will lead you. It is configured at the top of the gamebook file. Remember to tag this section correctly to be included. Nothing should lead to this section when building without the demo tag option. Go on to 5. 
4 (theend) -  This is the end. It is also included in the demo. 
5 (demo2) -  This is the second demo section. You can go on to the end at 4 or alternative (not in demo) end at 2. 
6 (notindemo2) -  This section is not included in the demo. You can go on to 5 or 4 or 2. 
7 (demo1) -  This is the first demo section (not counting the start section). You can go on to the end at 4. 
8 (notindemo1) -  This section is not included in the demo. Turn to 6. 
END DEBUG OUTPUT
