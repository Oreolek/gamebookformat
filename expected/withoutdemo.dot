digraph gamebook {

 1->8

1->5

1->7
3->5
5->2

5->4
6->4

6->5

6->2
7->4
8->6
}
