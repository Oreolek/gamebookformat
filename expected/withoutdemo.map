{"notindemo2": 6, "altend": 2, "demo2": 5, "start": 1, "demo1": 7, "theend": 4, "demoinfo": 3, "notindemo1": 8}