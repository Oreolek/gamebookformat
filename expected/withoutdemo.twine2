<tw-storydata name="Gamebook" startnode="1" creator="Twine" creator-version="2.0" format="Harlowe" options="">
<style role="stylesheet" id="twine-user-stylesheet" type="text/twine-css"></style><script role="script" id="twine-user-script" type="text/twine-javascript"></script>
<tw-passagedata pid="1" name="1" tags="" position="10,10"> This gamebook contains a few sections tagged as demo, that will be included when building the example as withdemo.gamebook (see withdemo.gamebook.options file). There are also some sections without the demo tag that will be ignored. This version is built without options and includes all sections. Go on to [[7]]
 or [[5]]
 or [[8]]
. </tw-passagedata>
<tw-passagedata pid="2" name="2" tags="" position="20,20"> Alternative end. Not included in the demo.</tw-passagedata>
<tw-passagedata pid="3" name="3" tags="" position="30,30"> This is where references to non-demo sections will lead you. It is configured at the top of the gamebook file. Remember to tag this section correctly to be included. Nothing should lead to this section when building without the demo tag option. Go on to [[5]]
. </tw-passagedata>
<tw-passagedata pid="4" name="4" tags="" position="40,40"> This is the end. It is also included in the demo. </tw-passagedata>
<tw-passagedata pid="5" name="5" tags="" position="50,50"> This is the second demo section. You can go on to the end at [[4]]
 or alternative (not in demo) end at [[2]]
. </tw-passagedata>
<tw-passagedata pid="6" name="6" tags="" position="60,60"> This section is not included in the demo. You can go on to [[5]]
 or [[4]]
 or [[2]]
. </tw-passagedata>
<tw-passagedata pid="7" name="7" tags="" position="70,70"> This is the first demo section (not counting the start section). You can go on to the end at [[4]]
. </tw-passagedata>
<tw-passagedata pid="8" name="8" tags="" position="80,80"> This section is not included in the demo. Turn to [[6]]
. </tw-passagedata>
</tw-storydata>
