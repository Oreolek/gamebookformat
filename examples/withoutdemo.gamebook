missingto = demoinfo

* 1 start :demo:
This gamebook contains a few sections tagged as demo, that will
be included when building the example as withdemo.gamebook
(see withdemo.gamebook.options file). There are also some sections
without the demo tag that will be ignored. This version is built
without options and includes all sections.
Go on to [[demo1]] or [[demo2]] or [[notindemo1]].

* notindemo1
This section is not included in the demo. Turn to [[notindemo2]].

* actuallynotindemoeither :demo:notthisone:
Tagged as demo, but excluded anyay because of the --exclude flag given.

* notatall :notthisone:
Not included at all, because excluded by command-line, and not
demo-tagged either.

* demo1 :demo:
This is the first demo section (not counting the start section).
You can go on to the end at [[theend]].

* notindemo2
This section is not included in the demo. You can go on to
[[demo2]] or [[theend]] or [[altend]].

* demo2 :demo:
This is the second demo section.
You can go on to the end at [[theend]] or alternative (not in demo)
end at [[altend]].

* theend :demo:
This is the end. It is also included in the demo.

* demoinfo :demo:
This is where references to non-demo sections will lead you.
It is configured at the top of the gamebook file.
Remember to tag this section correctly to be included.
Nothing should lead to this section when building without
the demo tag option.
Go on to [[demo2]].

* altend
Alternative end. Not included in the demo.
