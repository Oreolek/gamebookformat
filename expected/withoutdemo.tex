\documentclass[a5paper,onecolumn]{book}

\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[hidelinks]{hyperref}
\usepackage{graphicx}

\usepackage[top=3.3cm, bottom=3.3cm, left=2cm, right=2cm]{geometry}
\newif\ifpdf
\ifx\pdfoutput\undefined
  \pdffalse
\else
  \ifnum\pdfoutput=1
    \pdftrue
  \else
    \pdffalse
  \fi
\fi

\title{Gamebook}
\author{}
\date{}

\newcounter{sectionnr}

\begin{document}

\maketitle

\clearpage

\thispagestyle{empty}

\pagestyle{empty}


Turn to 1 to begin.
\phantomsection
\refstepcounter{sectionnr}
\label{section1}
\subsection*{\begin{center} \textbf{1} \end{center}}

 \noindent
 This gamebook contains a few sections tagged as demo, that will be included when building the example as withdemo.gamebook (see withdemo.gamebook.options file). There are also some sections without the demo tag that will be ignored. This version is built without options and includes all sections. Go on to \textbf{\autoref{section7}} or \textbf{\autoref{section5}} or \textbf{\autoref{section8}}. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section2}
\subsection*{\begin{center} \textbf{2} \end{center}}

 \noindent
 Alternative end. Not included in the demo.
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section3}
\subsection*{\begin{center} \textbf{3} \end{center}}

 \noindent
 This is where references to non-demo sections will lead you. It is configured at the top of the gamebook file. Remember to tag this section correctly to be included. Nothing should lead to this section when building without the demo tag option. Go on to \textbf{\autoref{section5}}. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section4}
\subsection*{\begin{center} \textbf{4} \end{center}}

 \noindent
 This is the end. It is also included in the demo. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section5}
\subsection*{\begin{center} \textbf{5} \end{center}}

 \noindent
 This is the second demo section. You can go on to the end at \textbf{\autoref{section4}} or alternative (not in demo) end at \textbf{\autoref{section2}}. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section6}
\subsection*{\begin{center} \textbf{6} \end{center}}

 \noindent
 This section is not included in the demo. You can go on to \textbf{\autoref{section5}} or \textbf{\autoref{section4}} or \textbf{\autoref{section2}}. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section7}
\subsection*{\begin{center} \textbf{7} \end{center}}

 \noindent
 This is the first demo section (not counting the start section). You can go on to the end at \textbf{\autoref{section4}}. 
\vspace{1em}
\phantomsection
\refstepcounter{sectionnr}
\label{section8}
\subsection*{\begin{center} \textbf{8} \end{center}}

 \noindent
 This section is not included in the demo. Turn to \textbf{\autoref{section6}}. 
\vspace{1em}
\end{document}
